docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+Steps to Debug WSO2 Identity Server Development Environment in Eclipse:
+
+All WSO2 products are based on a common platform called Carbon. It is therefore essential to obtain and build this layer before attempting to build the desired product itself. The build process can be divided into three distinct steps:
+1.	Build the carbon-kernel code.
+2.	Build the carbon-identity-framework code.
+3.	Build the WSO2 identity server code.
```

Comment:
Change 'WSO2 identity server' to 'WSO2 Identity Server'. Apply elsewhere if applicable.